pinot-controller/src/main/resources/webapp/js/init.js

```diff
@@ -105,10 +105,15 @@ var HELPERS = {
   },
 
   executeQuery: function(query, traceEnabled, callback) {
-    var url = "/pql?pql=" + encodeURIComponent(query) +"&trace=" + traceEnabled;
+    var url = "/pql";
+    var params = JSON.stringify({
```

Comment:
can you please add an option to enable/disable starTree

Reply:
I will address starTree with another pr.